webviz_subsurface/plugins/_inplace_volumes_onebyone.py

```diff
@@ -549,17 +603,29 @@ def calculate_table_rows(df, response):
                 {
                     "Sens Name": str(sensname),
                     "Sens Case": str(senscase),
-                    "Minimum": f"{values.min():.2e}",
-                    "Maximum": f"{values.max():.2e}",
-                    "Mean": f"{values.mean():.2e}",
-                    "Stddev": f"{values.std():.2e}",
-                    "P10": f"{np.percentile(values, 90):.2e}",
-                    "P90": f"{np.percentile(values, 10):.2e}",
+                    "Minimum": values.min(),
+                    "Maximum": values.max(),
+                    "Mean": values.mean(),
+                    "Stddev": values.std(),
+                    "P10": np.percentile(values, 90),
+                    "P90": np.percentile(values, 10),
                 }
             )
         except KeyError:
             pass
-    return table
+    columns = [
+        {**{"name": i[0], "id": i[0]}, **i[1]}
+        for i in InplaceVolumesOneByOne.TABLE_STATISTICS
+    ]
+    for col in columns:
+        try:
+            col["format"]["locale"]["symbol"] = [
+                "",
+                f"{VOLUME_UNITS.get(response, '')}",
+            ]
```

Comment:
for now, the "symbol" list is overwritten as long as it exists, removing the option of having anything in front of the number, could change to only overwriting the second element of the "symbol" list (which is the unit after the si-prefix) if you think it is preferable